### DISCUSSION TITLE
打开以后一直显示null

### DISCUSSION THREAD

**Discussion by meilin17** [windingwind/zotero-better-notes#1013] [Q&A]
![0fd353d425b39c01a6ff4703ab53baa](https://github.com/windingwind/zotero-better-notes/assets/174098124/ef51b88b-9f63-4b8b-975d-b23403c6993f)

**Comment by windingwind** [comment]
请详细描述问题

**Reply by meilin17** [reply]
打开插件以后工作区一直显示null，无法建立笔记

**Reply by windingwind** [reply]
据你描述符合bug，请按issue模板报告bug。此处关闭

**Reply by windingwind** [reply]
https://github.com/windingwind/zotero-better-notes/issues
